R/runner.R

```diff
@@ -145,11 +163,15 @@ mcstate_run_chain <- function(pars, model, sampler, n_steps, rng,
   ## surface to the user in the final object (or summarise them in
   ## some particular way with no guarantees about the format).  We
   ## might hold things like start and stop times here in future.
-  internal <- list(used_r_rng = !identical(get_r_rng_state(), r_rng_state),
-                   rng_state = rng$state(),
-                   sampler_state = sampler$get_internal_state())
+  internal <- list(
+    used_r_rng = !identical(get_r_rng_state(), r_rng_state),
+    state = list(
+      chain = state,
```

Comment:
Could consider rename `state` to `chain_state` here and on 142 ? Just because of aesthetics of state on 168